iCub_manual/conf_manual/iCubGazeboV3_visuomanip/gazebo_icub_right_leg.ini

```diff
@@ -0,0 +1,73 @@
+[include "gazebo_icub_robotname.ini"]
+
+[WRAPPER]
+# name of the wrapper device to be instatiated by the factory
+device controlboardwrapper2
+# rate of output streaming from ports in ms
+period 10
+# output port name (a leading '/' will be added by code)
+name /${gazeboYarpPluginsRobotName}/right_leg
+# Total number of joints
+joints 6
+# list of MotorControl device to use
+networks ( right_leg )
+# for each network specify the joint map
+right_leg 0 5 0 5
+# Verbose output (on if present, off if commented out)
+#verbose
+
+[TRAJECTORY_GENERATION]
+trajectory_type minimum_jerk
+
+# Specify configuration of MotorControl devices
+[right_leg]
+# name of the device to be instatiated by the factory
+device gazebo_controlboard
+#jointNames list
+jointNames r_hip_pitch r_hip_roll r_hip_yaw r_knee r_ankle_pitch r_ankle_roll
+name right_leg
+
+min_stiffness 0.0    0.0    0.0    0.0    0.0    0.0
+max_stiffness 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0
+min_damping   0.0    0.0    0.0    0.0    0.0    0.0
+max_damping   100.0  100.0  100.0  100.0  100.0  100.0
+
+#PIDs:
+# this information is used to set the PID values in simulation for GAZEBO, we need only the first three values
+[POSITION_CONTROL]
+controlUnits  metric_units
+controlLaw    joint_pid_gazebo_v1
+kp            17.453 17.453 17.453 17.453 17.453 17.453
+kd            0.174  0.174  0.174  0.174  0.174  0.174
+ki            0.174  0.174  0.174  0.174  0.174  0.174
+maxInt        9999   9999   9999   9999   9999   9999
+maxOutput     9999   9999   9999   9999   9999   9999
+shift         0.0    0.0    0.0    0.0    0.0    0.0
+ko            0.0    0.0    0.0    0.0    0.0    0.0
+stictionUp    0.0    0.0    0.0    0.0    0.0    0.0
+stictionDwn   0.0    0.0    0.0    0.0    0.0    0.0
+
+[VELOCITY_CONTROL]
+velocityControlImplementationType integrator_and_position_pid
+controlUnits  metric_units
+controlLaw    joint_pid_gazebo_v1
+kp            8.726 8.726 8.726 8.726 8.726 8.726
+kd            0.349 0.349 0.349 0.349 0.349 0.349
+ki            0.176 0.176 0.176 0.176 0.176 0.176
+maxInt        9999  9999  9999  9999  9999  9999
+maxOutput     9999  9999  9999  9999  9999  9999
+shift         0.0   0.0   0.0   0.0   0.0   0.0
+ko            0.0   0.0   0.0   0.0   0.0   0.0
+stictionUp    0.0   0.0   0.0   0.0   0.0   0.0
+stictionDwn   0.0   0.0   0.0   0.0   0.0   0.0
+
+[IMPEDANCE_CONTROL]
+controlUnits  metric_units
+controlLaw    joint_pid_gazebo_v1
+stiffness     0.0   0.0   0.0   0.0   0.0   0.0
+damping       0.0   0.0   0.0   0.0   0.0   0.0
+
+[LIMITS]
+jntPosMax 85.0 90.0 70.0 0.0 30.0 20.0
+jntPosMin -35.0 -15.0 -70.0 -100.0 -30.0 -20.0
+jntVelMax 100.0 100.0 100.0 100.0 100.0 100.0
```

Comment:
If I am not wrong, this file is unused. Can we delete it?

Reply:
indeed you are correct. Left-over files from copy/paste.  Fixed with 494367d2c695608b9b140170a20d225843dec4f3